insights/parsers/krb5.py

```diff
@@ -0,0 +1,151 @@
+"""
+krb5 Configuration file - File
+==============================
+krb5 Configuration are ``/etc/krb5.conf`` and ``/etc/krb5.conf.d/*``,
+and the content format is similar to ``INI config``, but they include
+values that span multiple lines. Multi-line values start with a '{'
+and end with a '}', and we join them together by setting the `is_squ`
+variable to True while in a multi-line value.
+
+The content of file ``/etc/krb5.conf`` and ``/etc/krb5.conf.d/*`` is
+recorded via INI format.
```

Comment:
You might consider rewording this since you've already described the INI format above, for example:
```
Sample content of ``/etc/krb5.conf`` and ``/etc/krb5.conf.d/*`` files 
is shown in the *Examples* section below.
```